.travis/deploy_develop_to_openshift.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_BRANCH" == “develop" ]; then
+bash <(curl -s https://codecov.io/bash) -f '!*datasafe-business*' -F unittests
```

Comment:
These:
```
bash <(curl -s https://codecov.io/bash) -f '!*datasafe-business*' -F unittests 
bash <(curl -s https://codecov.io/bash) -f '*datasafe-business*' -F e2e_tests
```
Should not be here, but rather executed each build - placed in after_success (maybe in separate bash script), **but** they need fixes so that they do pass correct branch, so that if: 
- branch is develop - codecov receives develop in VCS_BRANCH_NAME
- it is PR - it receives PR branch name:
https://docs.travis-ci.com/user/environment-variables/
TRAVIS_PULL_REQUEST_BRANCH

Reply:
updated as per the requirement.